.travis.yml

```diff
@@ -6,11 +6,7 @@ go:
   - 1.6
 
 before_install:
-  - go get github.com/golang/mock/gomock
-  - go get github.com/golang/mock/mockgen
-  - go get github.com/onsi/gomega
-  - go get github.com/onsi/ginkgo
-  - go get golang.org/x/tools/cmd/cover
+  - make dependencies
```

Comment:
Arg. that really ought to work...

Did you try go get -t scootdev/scoot/... ? go get is supposed to get dependencies so that should be enough...

Oh wait! Maybe it's because mockgen isn't actually listed as a dependency? What if you add:

import (
  _ "path/to/mockgen"
)
to sagalog.go (so that go install will install it)?